sapl/sessao/tests/test_sessao.py

```diff
@@ -170,3 +173,130 @@ def registro_votacao_com(ordem, expediente):
+                                        nome='ok',
+                                        natureza="A")
+    registro = mommy.make(RegistroVotacao,
+                          tipo_resultado_votacao=tipo_resultado_votacao,
+                          materia=materia,
+                          expediente=expediente)
+    presenca = mommy.make(SessaoPlenariaPresenca,
+                          sessao_plenaria=sessao_plenaria)
+    parlamentar = mommy.make(Parlamentar)
+    voto_parlamentar = mommy.make(VotoParlamentar,
+                                  votacao=registro,
+                                  parlamentar=parlamentar,
+                                  expediente=expediente)
+    sessao_plenaria.delete()
+    expediente_filter = ExpedienteMateria.objects.filter(
+                            sessao_plenaria=sessao_plenaria).exists()
+    registro_filter = RegistroVotacao.objects.filter(
+                        tipo_resultado_votacao=tipo_resultado_votacao,
+                        materia=materia,
+                        expediente=expediente).exists()
+    presenca_filter = SessaoPlenariaPresenca.objects.filter(
+                        sessao_plenaria=sessao_plenaria).exists()
+    parlamentar_filter = Parlamentar.objects.exists()
+    voto_parlamentar_filter = VotoParlamentar.objects.filter(
+                                    expediente=expediente).exists()
+    assert registro_filter == False
+    assert expediente_filter == False
+    assert presenca_filter == False
+    assert voto_parlamentar_filter == False
+    assert parlamentar_filter == True #  Não exclui Parlamentar
+
+
+@pytest.mark.django_db(transaction=False)
+def test_delete_sessao_plenaria_cascade_integrante_mesa():
+    sessao_plenaria = create_sessao_plenaria()
+    mesa = mommy.make(IntegranteMesa,
+                      sessao_plenaria=sessao_plenaria)
+    sessao_plenaria.delete()
+    mesa_filter = IntegranteMesa.objects.filter(
+                        sessao_plenaria=sessao_plenaria).exists()
+    assert mesa_filter == False
+
+@pytest.mark.django_db(transaction=False)
+def test_delete_sessao_plenaria_cascade_expedientesessao():
+    sessao_plenaria = create_sessao_plenaria()
+    expediente_sessao = mommy.make(ExpedienteSessao,
+                      sessao_plenaria=sessao_plenaria)
+    sessao_plenaria.delete()
+    expediente_sessao_filter = ExpedienteSessao.objects.filter(
+                                    sessao_plenaria=sessao_plenaria).exists()
+    assert expediente_sessao_filter == False
+
+@pytest.mark.django_db(transaction=False)
+def test_delete_sessao_plenaria_cascade_orador():
+    sessao_plenaria = create_sessao_plenaria()
+    expediente_sessao = mommy.make(Orador,
+                      sessao_plenaria=sessao_plenaria)
+    sessao_plenaria.delete()
+    orador_filter = Orador.objects.filter(
+                            sessao_plenaria=sessao_plenaria).exists()
+    assert orador_filter == False
```

Comment:
Desculpa minha chatice, mas uma forma "pythonica" seria 
```suggestion
    assert not orador_filter
```